pallets/pools/src/tranche.rs

```diff
@@ -1453,6 +1490,40 @@ where
 	Ok(risk_buffers)
 }
 
+pub(crate) fn create_asset_metadata<Balance, CurrencyId>(
+	decimals: u32,
+	currency: CurrencyId,
+	parachain_id: ParaId,
+	token_name: Vec<u8>,
+	token_symbol: Vec<u8>,
+) -> Result<AssetMetadata<Balance, CustomMetadata>, DispatchError>
```

Comment:
It seems like this never fails so this could return just `AssetMetadata<Balance, CustomMetadata>`, right?